cloud

```diff
@@ -1243,6 +1243,22 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+# Deploy a Google Cloud Storage Bucket
+# Args: project storage_bucket
+function storage_deploy() {
+    declare -r project="$1"; shift
+    declare -r storage_bucket="$1"; shift
+    gsutil mb -p "$project" -c STANDARD -l "us-central1" "gs://$storage_bucket"
+    gsutil iam ch allUsers:objectViewer "gs://$storage_bucket/"
```

Comment:
Please stop these commands generating output, like "bucket created" and "bucket removed".

Reply:
Use the `-q` option to `gsutil`. Like this: `gsutil -q mb blah blah`.